### DISCUSSION TITLE
java8下搭建rocketMQ，broker_default.log 报instrumneted a sprcial java.util.Set into : sun.nio.ch.EPollSelectorImpl,jps有namesrv和proxy

### DISCUSSION THREAD

**Discussion by alloyinfo** [apache/rocketmq#7291] [Q&A]
java8下搭建rocketMQ，broker_default.log 报instrumneted a sprcial java.util.Set into : sun.nio.ch.EPollSelectorImpl,jps有namesrv和proxy

**Comment by mxsm** [comment]
@alloyinfo能贴出日志中的错误日志堆栈的信息吗？

**Reply by alloyinfo** [reply]
INFO main - instrumented a special java.util.Set into: sun.nio.ch.EPollSelectorImpl@4eb386

打印的info日志

**Reply by alloyinfo** [reply]
没有堆栈

**Comment by joeCarf** [comment]
使用的是哪个版本呢

**Comment by HappyAxin** [comment]
5.1.4 同样有这个问题，楼主解决了吗？

**Comment by figuewang** [comment]
如果是idea编译器的话 把jdk统一设置一下，感觉是jdk不统一，instrumneted校验是jdk11才加的，你应该不是jdk8
